web_src/js/features/user-auth-webauthn.js

```diff
@@ -113,10 +113,10 @@ function webauthnRegistered(newCredential) {
 
 function webAuthnError(errorType, message) {
   $('#webauthn-error [data-webauthn-error-msg]').hide();
-  if (errorType === 0 && message && message.length > 1) {
-    $(`#webauthn-error [data-webauthn-error-msg=0]`).text(message);
-    $(`#webauthn-error [data-webauthn-error-msg=0]`).show();
-  } else {
+  if (errorType === 'general' && message && message.length > 1) {
+    $(`#webauthn-error [data-webauthn-error-msg=general]`).text(message);
+    $(`#webauthn-error [data-webauthn-error-msg=general]`).show();
```

Comment:
```suggestion
  if (errorType === 'general') {
    $(`#webauthn-error [data-webauthn-error-msg=general]`).text(message || 'unknown error');
    $(`#webauthn-error [data-webauthn-error-msg=general]`).show();
```

Otherwise if `errorType === 'general' && !message`, user would see the unrelated error messages.

Reply:
No they wouldn't look at the next else but your suggestion is better.